docs/software-design-federation-gateway-service.md

```diff
@@ -133,7 +133,7 @@ These key-value-pairs can be followed by additional attributes. The additional a
 | **Batching Service**
 | Batch Process started | INFO | Batch process started | n/a |
 | Batch Process: Created Batch | INFO | Batch created | batchTag, diagnosisKeyCount |
```

Comment:
This message could documented as well: 
| Batch Process: Successful finish | INFO | successfully finish the document batching process - no more unprocessed diagnosis keys left | n/a |